helpers/mobileLogin.ts

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+import { Alert } from 'react-native';
+
+export default async function mobileLogin(email, password) {
```

Comment:
Lol "I heard you like keywords so I put keywords in your keywords"

Reply:
export default async function await fetch then catch